native/android/app/src/main/java/tuerantuer/app/integreat/MainApplication.java

```diff
@@ -9,6 +9,8 @@
 import com.facebook.react.ReactPackage;
 import com.facebook.soloader.SoLoader;
 import java.lang.reflect.InvocationTargetException;
+import com.facebook.react.bridge.JSIModulePackage;
+import com.swmansion.reanimated.ReanimatedJSIModulePackage;
```

Comment:
Needed to get Bottom Action Sheet that is using reanimated to work

Reply:
But they are just imported and not used in this java file.

Reply:
They are used :)

Reply:
dunno why these changes are displayed. i merged main branch maybe after creating pr